contracts/README.md

```diff
@@ -1,7 +1,7 @@
 # EigenDA Contracts
 This package contains all smart contracts used to power EigenDA's on-chain operations. This includes both core protocol logic and verification constructs that a rollup can leverage to verify certificate integrity. This project uses both NPM and local submodules for dependency management. Most recently published NPM release artifacts can be found [here](https://www.npmjs.com/package/@eigenda/contracts).
 
-This project is divided into core and periphery. Versions in core represent changes in the eigenDA protocol, while versions in periphery/cert represent changes in an eigenDA blob verification certificate.
+This project is divided into core and integrations. Versions in core represent changes in the eigenDA protocol, while versions in integrations/cert represent changes in an eigenDA blob verification certificate.
```

Comment:
```suggestion
This project is divided into core and integrations. Versions in core represent changes in the eigenDA protocol, while versions in integrations/cert represent changes in eigenDA Cert verification contracts.
```

Reply:
cert TYPE versions. updated here https://github.com/Layr-Labs/eigenda/pull/1667/commits/3c7fc8e9eb09d544d73013487350f1432ea11f17

Reply:
> represent changes in EigenDA blob verification certificate types

Is it really just certificate *types*? `integrations/cert` appears to contain not only cert types, but also all cert related contracts.

Reply:
@pakim249CAL

Reply:
I see, was just a bit confused by the statement. Makes sense now, though